backend/server.js

```diff
@@ -1,7 +1,14 @@
 import express from 'express';
+import mongoose from 'mongoose';
 import data from './data.js';
+import userRouter from './routers/userRouter.js';
 
 const app = express();
+mongoose.connect(process.env.MONGODB_URL || 'mongodb://localhost/amazona', {
+  useNewUrlParser: true,
```

Comment:
1)go to the control panel

2)Select System and Security and choose the last option: Administrative tools.

3)Select Services

4)Search for MongoDB Server (MongoDB)

5) Right-click on it and click on the run.

6)and close the Services blindly

7)last but not least, change your MongoDB URL to mongodb://0.0.0.0:27017/

8)Restart your server and that worked for me!